src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -333,11 +334,6 @@ public override IOperation VisitAnonymousFunction(IAnonymousFunctionOperation op
             return new AnonymousFunctionExpression(operation.Symbol, Visit(operation.Body), ((Operation)operation).SemanticModel, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
         }
 
-        public override IOperation VisitFlowAnonymousFunction(IFlowAnonymousFunctionOperation operation, object argument)
-        {
```

Comment:
>operation.Id.Id [](start = 44, length = 15)

Should probably take CaptureId. I'll follow up on that myself.

Reply:
Yes, I have a prototype comment on CaptureId.Id - we should make that a private field.

---
In reply to: [192252108](https://github.com/dotnet/roslyn/pull/27254#discussion_r192252108) [](ancestors = 192252108)